src/main/java/org/dataone/bookkeeper/resources/UsagesResource.java

```diff
@@ -104,20 +107,22 @@ public UsagesResource(Jdbi database, DataONEAuthHelper dataoneAuthHelper) {
         List<String> subjects = new ArrayList<>();
         Boolean isProxy = isAdmin && requestor != null;
 
-        // Admin users can make request as another user
-        if(requestor != null) {
-            if(isAdmin) {
-                caller.setSubject(requestor);
-            } else {
-                throw new WebApplicationException(caller.getSubject() + " does not have admin privilege needed to set 'requestor'. ", Response.Status.FORBIDDEN);
-            }
-        }
-
         try {
+            // Admin users can make request as another user
+            if(requestor != null) {
```

Comment:
Let's be consistent with `if` whitespace
```suggestion
            if (requestor != null) {
```